nntrainer/graph/graph_core.h

```diff
@@ -235,6 +233,20 @@ class GraphCore {
    */
   unsigned int getNumOutputNodes() const { return output_list.size(); }
 
+  /**
+   * @brief       replace output node
+   * @param idx   output node index to be replaced
+   * @param node  graph node shared pointer to replace
+   */
+  void replaceOutputNode(unsigned int idx, std::shared_ptr<GraphNode> node) {
+    output_list[idx] = node;
+  }
```

Comment:
I wonder if the size of ouptut_list is the same with graph node size according to line #293